tools/check_format.py

```diff
@@ -94,15 +93,20 @@ def checkFormatVisitor(arg, dir_name, names):
 
 
 if __name__ == "__main__":
-  if len(sys.argv) != 2 and len(sys.argv) != 3:
-    print(USAGE)
-    sys.exit(1)
-
-  operation_type = sys.argv[1]
-  target_path = sys.argv[2] if len(sys.argv) == 3 else "."
+  parser = argparse.ArgumentParser(description='Check or fix file format.')
+  parser.add_argument('operation_type', type=str, help="specify if the run should 'check' or 'fix' format.")
```

Comment:
nice. updated to use choices. I left it as an obligatory positional argument to minimize usage complexity with two optional positional arguments back to back. In truth this could all be optional arguments but I don't want to break anyone that might have scripts using this script with the old usage.